WooCommerce/Classes/POS/Presentation/PointOfSaleCollectCashView.swift

```diff
@@ -112,6 +117,46 @@ struct PointOfSaleCollectCashView: View {
     }
 }
 
+private extension PointOfSaleCollectCashView {
+    func parseCurrency(_ amountString: String) -> NSDecimalNumber? {
```

Comment:
I've attempted to implement this from zero using `Decimal` but dealing with currency was anything but fun, so I've relied on our own `CurrencyFormatter` implementation. That said we seem to carry our own downsides (eg: https://github.com/woocommerce/woocommerce-ios/issues/13829 ) so I'm not sure if there's a better way we could use to handle formatting here, since we use the same currency formatter in other places in POS 🤔

Reply:
Yes, understandable. It's okay to use what we currently have. We can make improvements internally within the formatters.

The largest issues I've seen were when converting amounts back and forth when creating and then editing Order fields again. It's not the case with this POS cash payment so it should be fine.